backend/infrahub/core/task/task.py

```diff
@@ -70,7 +72,7 @@ async def query(
                 logs = [
                     {
                         "node": await TaskLog.from_db(result, extras={"task_id": task_result.get("uuid")}).to_graphql(
-                            fields=log_fields
+                            fields=StandardNodeQueryFields(node=log_fields)
```

Comment:
This is just to make mypy happy for now. But this is just some old code that isn't being used and should be deleted.